autoload/airline/extensions/anzu.vim

```diff
@@ -7,7 +7,7 @@ function! airline#extensions#anzu#init(ext)
 endfunction
 
 function! airline#extensions#anzu#apply(...)
-	call airline#extensions#prepend_to_section("z", "%{anzu#search_status()}")
-" 	let w:airline_section_z = ' %{anzu#search_status()} ' . get(w:, "airline_section_z", "") . get(g:, "airline_section_z", "")
+	let g:anzu_status_format='[%i/%l]'
```

Comment:
I removed it. https://github.com/osyo-manga/vim-anzu/commit/5448ed48b64f9be06247be017a3341c4df24ab36